circuits/types/field/src/from_bits.rs

```diff
@@ -15,11 +15,12 @@
 // along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
 
 use super::*;
+use std::iter;
 
 impl<E: Environment> FromBits for Field<E> {
     type Boolean = Boolean<E>;
 
-    /// Initializes a new base field element from a list of little-endian bits *without* trailing zeros.
+    /// Initializes a new base field element from a list of little-endian bits.
```

Comment:
Please revert this comment

Reply:
I believe you added support for trailing zeros in this commit: https://github.com/AleoHQ/snarkVM/commit/ecce1aa57b50d48395d6db49a260e81b7dc681b0

Reply:
Sure, can you update it to enumerate the cases that this method supports then?